go-controller/cmd/ovn-kube-util/app/gateway_init.go

```diff
@@ -92,7 +92,7 @@ func initGateway(context *cli.Context) error {
 		return fmt.Errorf("One of physical-interface or bridge-interface has to be specified")
 	}
 
-	return util.GatewayInit(clusterIPSubnet, nodeName, physicalIP,
+	return util.GatewayInit([]string{clusterIPSubnet}, nodeName, physicalIP,
```

Comment:
hmm - I'm confused - why is this hardcoed to a list with one entry?

Reply:
@squeed this util function is called with an array in other places but in this context there is only a single element. Should I make the ovn-kube-utils support multiple subnets as well?

Reply:
no need - I just wanted to understand. If we need it, we can add it later.